internal/manifestvalues/cli/configure.go

```diff
@@ -129,7 +135,23 @@ func printHeader(name string, def v1alpha1.ValueDefinition) {
 	}
 	fmt.Fprintln(os.Stderr, bold(title))
 	if len(def.Metadata.Description) > 0 {
-		fmt.Fprintln(os.Stderr, def.Metadata.Description)
+		printMarkdown(os.Stderr, def.Metadata.Description)
+	}
+}
+
+func printMarkdown(w io.Writer, text string) {
+	md := goldmark.New(
+		goldmark.WithRenderer(renderer.NewRenderer(
+			renderer.WithNodeRenderers(
+				goldmarkutil.Prioritized(cliutils.MarkdownRenderer(), 1000),
+			),
+		)),
+	)
+	var buf bytes.Buffer
+	if err := md.Convert([]byte(text), &buf); err != nil {
+		util.Must(fmt.Fprintln(w, text))
+	} else {
+		util.Must(fmt.Fprintln(w, strings.TrimSpace(buf.String())))
```

Comment:
The old implementation would have an empty line here

```suggestion
		util.Must(fmt.Fprint(w, strings.TrimSpace(buf.String())+"\n\n"))
```